doc/value.md

```diff
@@ -0,0 +1,21 @@
+@property can-observation.prototype.value value
+@parent can-observation.prototype
+@description Reads the value of the observation.
+@signature `observation.value`
+
+The following creates a `fullName` observation that derives its values from
+the `person` observable. The value of the observation is read with
+`fullName.value`:
+
+```js
+import Observation from "can-observation";
+import observe from "can-observe";
+
+const person = observe( { first: "Grace", last: "Hopper" } );
+
+const fullName = new Observation( function() {
+	return person.first + " " + person.last;
+} );
+
+fullName.value; //-> "Grace Hopper";
+```
```

Comment:
Here’s what this page looks like:
<img width="681" alt="screen shot 2018-07-02 at 4 46 54 pm" src="https://user-images.githubusercontent.com/10070176/42191483-f2f205d6-7e17-11e8-8ad8-faf8ca3621e9.png">